addons/point_of_sale/models/pos_order.py

```diff
@@ -396,7 +415,7 @@ def create(self, values):
     @api.model
     def _complete_values_from_session(self, session, values):
         if values.get('state') and values['state'] == 'paid':
-            values['name'] = session.config_id.sequence_id._next()
+            values['name'] = self._compute_order_name()
```

Comment:
this is wrong, new method `_compute_order_name` expect instance of the the model not recordset but this method (`_complete_values_from_session`) is called from recordset, so self has no values or different values